src/components/Navbar.js

```diff
@@ -68,14 +59,17 @@ export class Navbar extends Component {
                   >
                     <i className="fa fa-user" />
                     <strong>
-                      {''} {localStorage.getItem('username') || this.props.user.username} {''}{' '}
+                      {''} {localStorage.getItem('username') || this.props.user.username}
+                      {''}
                     </strong>
                   </a>
-                  <div className="dropdown-menu" aria-labelledby="navbarDropdown" id="navbarDropdown">
+                  <div
+                    className="dropdown-menu"
+                    aria-labelledby="navbarDropdown"
+                    id="navbarDropdown"
+                  >
                     <a className="dropdown-item nav-link" href="!#" onClick={this.logoutUser}>
-                      <strong>
-                        Logout <i className="fa fa-sign-out" />
-                      </strong>
+                      Logout <i className="fa fa-sign-out" />
```

Comment:
Is this change connected to this ticket?

Reply:
Yeah, it is connected to the ticket as I made a change of instead of using the `<strong>` tag, I just executed it in sass.